cis_v300/section_1.sp

```diff
@@ -0,0 +1,152 @@
+  query         = query.azuread_guest_user_info
+  documentation = file("./cis_v300/docs/cis_v300_1_1_4.md")
+
+  tags = merge(local.cis_v300_1_1_common_tags, {
+    cis_item_id           = "1.1.4"
+    cis_level             = "1"
+    cis_type              = "manual"
+    microsoft_365_license = "E3"
+    service               = "Azure/ActiveDirectory"
+  })
+}
+
+benchmark "cis_v300_1_2" {
+  title         = "1.2 Teams & groups"
+  children = [
+    control.cis_v300_1_2_1
+  ]
+
+  tags = merge(local.cis_v300_1_2_common_tags, {
+    type    = "Benchmark"
+    service = "Azure/ActiveDirectory"
+  })
+}
+
+control "cis_v300_1_2_1" {
+  title         = "1.2.1 Ensure that only organizationally managed/approved public groups exist"
+  description   = "Microsoft 365 Groups is the foundational membership service that drives all teamwork across Microsoft 365. With Microsoft 365 Groups, you can give a group of people access to a collection of shared resources. While there are several different group types this recommendation concerns Microsoft 365 Groups. In the Administration panel, when a group is created, the default privacy value is `Public`."
+  query         = query.azuread_group_not_public
+  documentation = file("./cis_v300/docs/cis_v300_1_2_1.md")
+
+  tags = merge(local.cis_v300_1_2_common_tags, {
+    cis_item_id           = "1.2.1"
+    cis_level             = "2"
+    cis_type              = "automated"
+    microsoft_365_license = "E3"
+    service               = "Azure/ActiveDirectory"
+  })
+}
+
+benchmark "cis_v300_1_3" {
+  title         = "1.3 Settings"
+  children = [
+    control.cis_v300_1_3_1,
+    control.cis_v300_1_3_3
+  ]
+
+  tags = merge(local.cis_v300_1_3_common_tags, {
+    type    = "Benchmark"
+    service = "Azure/ActiveDirectory"
+  })
+}
+
+control "cis_v300_1_3_1" {
+  title         = "1.3.1 Ensure the 'Password expiration policy' is set to 'Set passwords to never expire'"
+  description   = "Microsoft cloud-only accounts have a pre-defined password policy that cannot be changed. The only items that can change are the number of days until a password expires and whether or whether passwords expire at all."
+  query         = query.azuread_user_password_not_set_to_expire
+  documentation = file("./cis_v300/docs/cis_v300_1_3_1.md")
+
+  tags = merge(local.cis_v300_1_3_common_tags, {
+    cis_item_id           = "1.3.1"
+    cis_level             = "1"
+    cis_type              = "automated"
+    microsoft_365_license = "E3"
+    service               = "Azure/ActiveDirectory"
+  })
+}
+
+control "cis_v300_1_3_3" {
+  title         = "1.3.3 Ensure 'External sharing' of calendars is not available"
+  description   = "External calendar sharing allows an administrator to enable the ability for users to share calendars with anyone outside of the organization. Outside users will be sent a URL that can be used to view the calendar."
+  query         = query.microsoft365_calendar_sharing_disabled
```

Comment:
@khushboo9024 Pls validate this 

```
1.3.3 Ensure 'External sharing' of calendars is not available .........................................................................................      
| |     ERROR: relation "microsoft365_my_calendar" does not exist (SQLSTATE 42P01)
```

Reply:
@rajlearner17 Please update the plugin and try again.